backend/typescript/models/pet.model.ts

```diff
@@ -0,0 +1,69 @@
+import {
+  Column,
+  Model,
+  Table,
+  ForeignKey,
+  BelongsTo,
+  DataType,
+} from "sequelize-typescript";
+import Animal_Type from "./animalType.model";
+// import { PetCareInfo } from "./PetCareInfo"
+
+import { Sex, PetStatus } from "../types";
+
+@Table({ timestamps: false, tableName: "pet" })
+export default class Pet extends Model {
+  @ForeignKey(() => Animal_Type)
+  @Column({})
+  animal_type_id!: number;
+
+  @BelongsTo(() => Animal_Type)
```

Comment:
Shouldn't the Animal_Type model have the BelongsTo association while the Pet model has the HasOne? Just wondering why you chose to use BelongsTo here

Reply:
No, the BelongTo association here implies that a Pet (specifically its animal_type_id) has one instance in the Animal Type table. Hence it belongs to that table. 

If we had HasMany here, it would imply that one Pet has many instances in Animal Type.

Does that make sense?